internal/gatewayapi/testdata/backendtlspolicy-multiple-targets.out.yaml

```diff
@@ -201,8 +205,6 @@ xdsIR:
                 name: policy-btls/envoy-gateway-ca
               sni: example.com
             weight: 1
-        directResponse:
```

Comment:
why did this change ?

Reply:
Fixes tests inputs in the Gateway API translator tests. Some tests were missing services and endpointSlices resources, which led to 500 errors in the output files. This PR adds the missing services and endpointSlices to ensure the expected behavior.